lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -695,7 +696,87 @@ def test_cohort_content(self):
             self.assertDictContainsSubset(
                 {'action_name': 'graded', 'attempted': 4, 'succeeded': 4, 'failed': 0}, result
             )
+        problem_names = [u'Homework 1: Problem - Pröblem0', u'Homework 1: Problem - Pröblem1']
+        header_row = [u'Student ID', u'Email', u'Username', u'Final Grade']
+        for problem in problem_names:
+            header_row += [problem + ' (Earned)', problem + ' (Possible)']
 
+        self.verify_rows_in_csv([
```

Comment:
@dan-f As I understood; What we can do is we can add a helper method and pass it the `user` and `grade` information; it can generate the expected rows to be matched with the generated rows. Then we can pass expected_rows to the `self.verify_rows_in_csv` which can compare it with the the recently generated csv. 

We can discuss if still needs any change.